tests/zfs-tests/tests/functional/cli_user/misc/misc.cfg

```diff
@@ -67,17 +67,17 @@ else
 	# zfs get/set subcommands - ordered as per the list above so we
 	# can iterate over both sets in an array
 	PROP_VALS="\
-		secure		discard		on		 \
+		passthrough	discard		on		 \
 		fletcher2	on				on \
 		on		legacy		none		on \
 		128k		none		on		on \
 		visible"
 
 	# these are an alternate set of property values
 	PROP_ALTVALS="\
-		noallow		noacl		off		 \
+		passthrough-x	noacl		off		 \
```

Comment:
Turns out that line last changed in 2013. Went ahead and pulled in the `groupmask` change to make it consistent.